static/sass/ds/7-utilities/_bg.scss

```diff
@@ -9,3 +9,17 @@
     background-color: $color-white-off;
   }
 }
+
+.has-white-off-bg-from-bp-l {
+  @include mq($from: bp-l) {
+    background-color: $color-white-off;
+  }
+}
+
+.has-bg-twitter {
+  background-color: $color-twitter;
+}
+
+.has-bg-facebook {
+  background-color: $color-facebook;
+}
```

Comment:
The good news is this is included in queso, so no need to add. The bad news is I failed to document it. I'll add it to the SCSS comments so it appears in the docs. I don't blame you for not seeing that one!

Reply:
Removed!